policies/rules.md

```diff
@@ -3,108 +3,115 @@ layout: default
 
 How do we limit the impact and costs associated with Social Isolation and Distancing, and ensure that we're ready to pick up where we left off when we come back together?
 
 Apps in this category might help connect people together, identify people in distress, facilitate remote working, on assist parents and educators continue classes.
 
-#### Scarcity & The Economy
+### Scarcity & the Economy
 
 Economics is the study of scarcity, and during the pandemic it's a struggle to understand the situation and how to most effectively use limited resources to maximum effect.
 
-How we better understand and manage the economic impacts, and use our resources most effectively to weather the largest economic change in modern history.
+How might we better understand and manage the economic impact, and use our resources most effectively, to weather the largest economic change in modern history?
 
-App in this category could include helping understand and manage changes in the work force, evaluate and track the effectiveness of government interventions, or better distribute scarce resources.
+Apps in this category could include helping understand and manage changes in the work force, evaluate and track the effectiveness of government interventions, or better distribute scarce resources.
 
-### Your Solution
+# Your Solution
 
 We believe faster, more effective and more available solutions are preferred to slower, less useful and limited ones.
 
-Therefore we will be judging projects on three main criteria:
+Therefore, we will be judging projects on three main criteria:
 
 1. **Effectiveness** - How well does this project address the problem it's attempting to solve?
 2. **Accessibility** - How easily are people are able to access this project?
 3. **Impact** - How many people have benefited from this project?
 
-### Your Team
+# Your Team
 
 Teams of up to 5 people are welcome to participate. For legal reasons we require that all team members be 13 years or older as of May 16th at 00:00 UTC if they wish to participate.
 
-### Weekly Competitions
+# Weekly Competitions
 
-For the majority of the event, teams may submit projects into a weekly competition. The winner is determined by a popular vote, and announced Tuesdays.
+For the majority of the event, teams may submit projects into a weekly competition. The winner is determined by a popular vote and announced Tuesdays.
 
 Teams must submit their apps by **Sunday, 23:59 UTC** and then the public has 24-hours from **Monday, 03:00 UTC** to **Tuesday, 03:00 UTC** to vote for their favourite. The winner will be announced by **Tuesday, 20:00 UTC**.
 
 _This is a great opportunity to promote your project and CODEVID-19 to friends, family, and your network!_
 
-[Submit your project for the weekly prizes](submissions.html#weekly-prize-submission)
+[View weekly submission details](submissions.html#weekly-prize-submission)
```

Comment:
I like the call to action we had before, asking them to view details sounds like it will link to the small print and not where to go to submit.

Reply:
Except the call to action was misleading.  I read the original and even thought we had links there to our Google Forms, but all we had were just more details.  The section we take them to does not allow them to submit their project at all, hence the renaming.

Reply:
I'll go with whichever your final decision on this is @MarkBennett

Reply:
Still waiting on your reply here @MarkBennett